src/eureka/lib/centerdriver.py

```diff
@@ -0,0 +1,106 @@
+# $Author: patricio $
+# $Revision: 304 $
+# $Date: 2010-07-13 11:36:20 -0400 (Tue, 13 Jul 2010) $
+# $HeadURL: file:///home/esp01/svn/code/python/branches/patricio/photpipe/lib/centerdriver.py $
+# $Id: centerdriver.py 304 2010-07-13 15:36:20Z patricio $
+
+import numpy as np
+from . import imageedit as ie
+from . import gaussian as g
+from ..S3_data_reduction import plots_s3
+
+
+def centerdriver(method, data, guess, trim, radius, size,
+                 mask=None, uncd=None, fitbg=1, maskstar=True,
+                 expand=5.0, psf=None, psfctr=None, i=None, m=None, meta=None):
+    """
+    Use the center method to find the center of a star in data, starting
+    from position guess.
+
+    Parameters:
+    -----------
```

Comment:
No colons after Parameters and Returns anywhere (and that also means shortening the underline below)